app/components/Reactions/assets/AddReactionEmoji.js

```diff
@@ -10,7 +10,7 @@ const AddReactionEmoji = ({ color }: Props) => (
   <div className={styles.container}>
     <span>
       <svg
-        ariaHidden="true"
+        aria-hidden="true"
```

Comment:
and this Warning annoyed the shit out me

Reply:
LOL yeah, this one's a goodamn bi***